_posts/2020-08-03-acidanthera-august.md

```diff
@@ -0,0 +1,155 @@
+* Fixed crashes when trying to read CLKT key
+
+#### [HibernationFixup 1.3.4](https://github.com/acidanthera/HibernationFixup)
+
+* Improve auto-hibernate feature: correct next wake time disregarding the current sleep phase.
+* Added constants for 11.0 support.
+
+#### [WhateverGreen 1.4.1](https://github.com/acidanthera/WhateverGreen/releases)
+
+* Added `igfxmetal=1` boot argument (and `enable-metal` property) to enable Metal on offline IGPU
+* Fixed applying patches on CometLake IGPUs, thx @apocolipse
+* Added constants required for 11.0 update
+* Added the use of RPS control for all the command streamers on IGPU (disabled via `igfxnorpsc=1`)
+* Add `-igfxvesa` to disable Intel Graphics acceleration.
+* Fix black screen on igfx since 10.15.5
+* Add workaround for rare force wake timeout panics on Intel KBL and CFL.
+* Add Intel Westmere graphics support.
+
+#### [Lilu 1.4.6](https://github.com/acidanthera/Lilu/releases)
+
+* Added preliminary definitions for 11.0 support
+* Temporarily disabled user patcher for 11.0
+* Added `external-audio` property to ignore PCI audio cards
+* Added in-memory symbol solving for 11.0
+* Fixed accidentally solving stabs instead of normal symbols
+* Added device publishing API to monitor device startup
+* Added DeviceInfo caching for improved performance
+* Added implicit slotted (medium) patches in KC mode to reduce patch size
+
+#### [CPUFriend 1.2.1](https://github.com/acidanthera/CPUFriend)
+
+* Added constants for 11.0 support
+
+#### [AirportBrcmFixup 2.0.8](https://github.com/acidanthera/AirportBrcmFixup)
+
+* Added constants for 11.0 support
+* Property 'pci-aspm-default' with value 0 is not required for Broadcom BCM4350 chipset (with non-apple subsystem-vendor-id), 
+since now it is injected/corrected and method IOPCIFamily::setASPMState called for provider to disable ASPM immediately.
+* Add required dependencies into OSBundleLibraries section
+* Remove injectors for AirPortBrcm4360 and AirPortBrcmNIC from main Info.plist and move them into separate plugins AirPortBrcm4360_Injector and 
+AirPortBrcmNIC_Injector (kexts with plist only).
+Under 10.16 (Big Sur) plugin AirPortBrcm4360_Injector.kext must be blocked by MaxKernel 19.9.9 or just removed, otherwise it will block loading of AirPortBrcmNIC
+since class AirPortBrcm4360 is unsupported.
+* Check whether brcmfx-driver value is incorrect (if specified value is unsupported in current osx system)
+* Support boot-arg and property `brcmfx-aspm` to override value used for pci-aspm-default 
+
+#### [DebugEnhancer 1.0.1](https://github.com/acidanthera/DebugEnhancer)
+
+* Added constants for 11.0 support
+
+#### [BrcmPatchRAM 2.5.4](https://github.com/acidanthera/BrcmPatchRAM)
+
+* Added inject BCM2070 - BCM943224HMB, BCM943225HMB Combo
+
+# Guide Updates
+
+Some of you may have noticed since our last update that the site has gone through a redesign and rewriting. The laptop guide has been merged and the post-install guide becoming a dedicated site.
```

Comment:
```suggestion
Some of you may have noticed since our last update that the site has gone through a redesign and rewrite. The laptop guide has been merged into the desktop guide to create a unified install guide and post-install has been separated to a dedicated site.
```